src/pages/api/updateFeedback.ts

```diff
@@ -18,15 +19,19 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
             req.socket.remoteAddress || ''
         );
     } catch {
-        return res.status(500).json({
+        res = res.status(500);
+        logger.error(logDataFromNextAPIReqAndRes(req, res));
```

Comment:
With the way that this is invoked, you are 1) building a log output separately and 2) logging that output with the function here.